test/tools/yulInterpreter/Interpreter.h

```diff
@@ -35,7 +35,19 @@ namespace yul
 namespace test
 {
 
-class InterpreterTerminated: dev::Exception
+class InterpreterTerminatedGeneric: dev::Exception
```

Comment:
```suggestion
class InterpreterTerminatedGeneric: public dev::Exception
```
(also for the others)